sdcm/nemesis.py

```diff
@@ -294,3 +294,30 @@ class ChaosMonkey(Nemesis):
     @log_time_elapsed
     def disrupt(self):
         self.call_random_disrupt_method()
+
+class UpgradeNemesis(Nemesis):
+
+    # upgrade a single node
+    def upgrade_node(self, node):
+        self.log.info('Upgrading a Node')
+        scylla_repo = get_data_path('scylla.repo.upgrade')
+        node.remoter.send_files(scylla_repo, '/etc/yum.repos.d/scylla.repo', verbose=True)
+        node.remoter.run('yum update')
```

Comment:
We probably want `yum upgrade -y scylla` here. Without the -y this command will prompt for user input.